locale/en.po

```diff
@@ -2167,19 +2167,23 @@ msgstr "Art Gallery"
 msgid "LICENSES"
 msgstr "Licenses"
 
-#: ../src/general/MainMenu.tscn:496
+#: ../src/general/MainMenu.tscn:420
+msgid "SUGGEST_A_FEATURE"
+msgstr "Suggest A Feature"
```

Comment:
A native speaker can correct me, but I think usually in titles small words like "a" are *not* capitalized. Also small words are not necessary to put in the translation keys, often their content is changed entirely. That's why I think the key should be visit suggestions site like I said before as then the English text can be replaced if we feel like it with something completely different.

Reply:
Sorry, I misread the last comment and only just noticed my mistake. Fixing it now